src/Makefile

```diff
@@ -76,17 +75,12 @@ $(CLEANDIRS):
 	$(MAKE) -s -C $(@:clean-%=%) clean
 
 phpvendors:
-	$(COMPOSERTMP)/composer install -q
-
-composer_download:
-	curl -sS https://getcomposer.org/installer | php
-	mkdir -p $(COMPOSERTMP)
-	mv composer.phar $(COMPOSERTMP)/composer
+	composer install -q
 
```

Comment:
Ok, that's a problem of the wrapper script. A correct altenrative would be to download composer to some temporary path (e.g. `tmp/composer/`) and later in the next step all the make command with the correct path (e.g. `PATH='tmp/composer/:$PATH make ....`).

Does this fit your 2 step approach?